terraform/variables.tf

```diff
@@ -0,0 +1,60 @@
+variable "app_name" {
+  description = "Application name"
+  type        = string
+  nullable    = false
+  default     = "cos-registration-server"
+}
+
+variable "channel" {
+  description = "Charm channel"
+  type        = string
+  nullable    = false
+  default     = "latest/edge"
```

Comment:
should we use latest/beta since we promoted it?

Reply:
Looking on charmhub, the latest relaese on `beta` dates back from april 2024 ...